Tasks/DownloadFileShareArtifactsV0/task.json

```diff
@@ -0,0 +1,80 @@
+{
+  "id": "E3CF3806-AD30-4EC4-8F1E-8ECD98771AA0",
+  "name": "DownloadFileShareArtifacts",
+  "friendlyName": "Download FileShare Artifacts",
+  "description": "Download FileShare Artifacts",
+  "helpMarkDown": "",
+  "category": "Utility",
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 0,
+    "Minor": 136,
+    "Patch": 0
+  },
+  "groups": [
+    {
+        "name": "advanced",
+        "displayName": "Advanced",
+        "isExpanded": false
+    }
+  ],
+  "demands": [],
+  "inputs": [
+    {
+      "name": "location",
+      "type": "string",
```

Comment:
Eventually we want this to probably come from a service endpoint where user can provide credentials to access as well. It will help for cross-platform scenarios like downloading on linux/mac. It doesn't have to be done now but if it will prevent us from doing a major version break later, it will be worth making this a service endpoint now. The Upload to Fileshare will use the same service endpoint.

Reply:
Yes, we had this discussion within the team and had decided on not adding this additional functionality at this point. This task will be used for implicit download in RM and currently we don't expect users to choose an endpoint during link time.